platform/export_inputs.go

```diff
@@ -97,7 +97,7 @@ func FillExportRunImage(i *LifecycleInputs, logger log.Logger) error {
 			if err != nil {
 				return err
 			}
-			if analyzedMD.RunImage == nil || analyzedMD.RunImage.Reference == "" {
```

Comment:
Isn't it still theoretically possible that this could be nil?

Reply:
no i don't think so -- it's not a pointer anymore, so it should always be either the "zero" values of that struct, or the filled-in values of that struct.